ecommerce/migrations/0038_rename_visible_in_bulk_form_product_is_private.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2.20 on 2023-08-31 11:25
+
+from django.db import migrations, models
+
+
+def invert_is_private(apps, schema_editor):
+    """
+    Inverts `Product.is_private` as we renamed `Product.visible_in_bulk_form` to `Product.is_private`.
+    """
+    Product = apps.get_model("ecommerce", "Product")
+    Product.objects.all().update(
+        is_private=models.Case(
+            models.When(is_private=False, then=models.Value(True)),
+            default=models.Value(False),
+        )
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("ecommerce", "0037_product_coupon_assignment_index"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="product",
+            old_name="visible_in_bulk_form",
+            new_name="is_private",
+        ),
+        migrations.AlterField(
+            model_name="product",
+            name="is_private",
+            field=models.BooleanField(
+                default=False,
+                help_text="Products can be Private or Public. Public products are listed in the product drop-down on the bulk purchase form at /ecommerce/bulk.",
+            ),
+        ),
+        migrations.RunPython(invert_is_private, reverse_code=migrations.RunPython.noop),
```

Comment:
I think we should write a script for revert migration as well. That would potentially be a call to invert function again i think.